src/Form.php

```diff
@@ -307,6 +307,9 @@ public function onSubmit(\Closure $callback)
             } catch (\Atk4\Data\ValidationException $val) {
                 $response = [];
                 foreach ($val->errors as $field => $error) {
+                    if (!$field) {
+                        throw ($val);
```

Comment:
```suggestion
            } catch (\Atk4\Data\ValidationException $e) {
                $response = [];
                foreach ($val->errors as $field => $error) {
                    if (!$field) {
                        throw $e;
```

Reply:
missing change for $val->errors - I will update